presignedcloudfronturl_tests.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
I don't think this file needs to exist. The testing of signed files should be generic regardless of how they are signed so the existing page should be sufficient